command/genesis/genesis.go

```diff
@@ -192,11 +192,11 @@ func setFlags(cmd *cobra.Command) {
 			contracts.ContractsRootFolder,
 			"the smart contracts folder",
 		)
-		cmd.Flags().BoolVar(
-			&params.bridgeEnabled,
+		cmd.Flags().StringVar(
+			&params.bridgeJSONRPCAddr,
 			bridgeFlag,
-			defaultBridge,
-			"enabling bridge, default is not enabled",
+			"",
```

Comment:
Can we setup here the default address?

Reply:
No I won't say so, because if `--bridge-json-rpc` not provided, it means that we are creating genesis in a "non-bridge-mode" (this flag replaced the previously used `--bridge`).

Reply:
* `--bridge-json-rpc` present without value, bridge enabled with default address, value provided in other case.
* Flag not provided bridge disabled.

Can we do that?

Reply:
Seems we cannot.

I've tried running genesis command like this:
`go run . genesis --block-gas-limit 10000000 --validator-set-size=4 --consensus polybft --epoch-size 5 --bridge-json-rpc`
and it returns a following error:

> flag needs an argument: --bridge-json-rpc

Tried this as a workaround (https://stackoverflow.com/a/70186443), but no luck as well :thinking: :
`cmd.Flags().Lookup(bridgeFlag).NoOptDefVal = "http://127.0.0.1:8545"`

Reply:
Fixed [fb9abbe](https://github.com/0xPolygon/polygon-edge/pull/989/commits/fb9abbe8661f3f160cba5dd828b5fc3b14e542bf)